bundles/persistence/org.openhab.persistence.dynamodb/java/org/openhab/persistence/dynamodb/internal/DynamoDBItem.java

```diff
@@ -48,6 +50,10 @@
 
     Date getTime();
 
+    Long getExpiration();
+
+    void setExpiration(long expirationTime);
```

Comment:
Please rename getter/setter and setter argument name to make it clear whether we have milliepoch or epoch-seconds